src/import-service/serverless.yml

```diff
@@ -20,6 +27,41 @@ provider:
         - 's3:*'
       Resource:
         - 'arn:aws:s3:::macbook-shop-uploaded/*'
+    - Effect: 'Allow'
+      Action:
+        - 'sqs:*'
+      Resource:
+        - Fn::GetAtt: [ SQSQueue, Arn ]
+    # - Effect: 'Allow'
+    #   Action:
+    #     - 'sns:*'
+    #   Resource:
+    #     - Ref: SNSTopic
+
+resources:
+  Resources:
+    SQSQueue:
```

Comment:
On the course architecture diagram SQS is displayed as part of the product-service

Reply:
For educational purposes to understand the differences